code/modules/surgery/encased.dm

```diff
@@ -12,15 +12,13 @@
 		TOOL_SAW = 100,
 		/obj/item/hatchet = 90,
 		/obj/item/chainsaw = 90,	// Chainsaws were actually invented to replace manual bonesaws. Obvously *these* chainsaws aren't exactly designed for that, but then again, neither is a hatchet or a set of wirecutters.
-		/obj/item/butcher_chainsaw = 90,
```

Comment:
Removing the chainsaw as valid surgery tools here isn't mentioned in the PR description or changelog. Does this no longer work as a result of the refactoring?